app/bundles/CoreBundle/Tests/DependencyInjection/Compiler/SmsTransportPassTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/*
+ * @copyright   2018 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ * @author      Jan Kozak <galvani78@gmail.com>
+ */
+
+namespace Mautic\CoreBundle\DependencyInjection\Compiler;
+
+use Mautic\CoreBundle\Test\AbstractMauticTestCase;
+use Mautic\SmsBundle\Sms\TransportChain;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class SmsTransportPassTest extends AbstractMauticTestCase
+{
+    public function testProcess()
+    {
+        $container = new ContainerBuilder();
+        $container->addCompilerPass(new SmsTransportPass());
+        $container
+            ->register('foo')
+            ->setPublic(true)
+            ->setAbstract(true)
+            ->addTag('mautic.sms_transport', ['alias'=>'fakeAliasDefault', 'fakeIntegrationDefault'])
+        ;
```

Comment:
Lets write the semicolons on the last line of code like the rest of the Mautic's code.